_episodes/14-supplemental-vscode.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Supplemental: Using Git from VS-Code"
+teaching: 10
+exercises: 0
+questions:
+- "How can I use Git with VS-Code?"
+objectives:
+- "Understand how to use Git from VS-Code."
```

Comment:
We could expand the objectives to something like:
- Understand the purpose and advantages of an integrated development environment (IDE).
- See how Git can be used within the workflow of an IDE.
- See how Git is set up in Visual Studio Code VS-Code.

Reply:
Once again I am going to disagree. We are showing one specific instance, these instructions don't apply to IDE's in general and we dont mention other IDE's. So we should not advertise as such in the objectives.

Reply:
yea na i wrong